Connectors/Rhino/Speckle.Connectors.RhinoShared/HostApp/Properties/PropertiesExtractor.cs

```diff
@@ -13,16 +13,21 @@ public class PropertiesExtractor
   {
     Dictionary<string, object?> properties = new();
     var userStrings = rhObject.Attributes.GetUserStrings();
-    foreach (var key in userStrings.AllKeys)
+    foreach (string? key in userStrings.AllKeys)
     {
+      if (key == null)
+      {
+        continue;
+      }
+
       try
       {
         if (key == "$block-instance-original-object-id$") // skip: this seems to be an invisible user string that shows up on block instances
         {
           continue;
         }
 
-        if (userStrings[key].StartsWith("%<"))
+        if (userStrings[key]?.StartsWith("%<") ?? false)
```

Comment:
Rhino.Common 8.21 nuget (used only by the importer) has more nullability decoration, so I need to handle it.